Podfile

```diff
@@ -53,6 +53,12 @@ post_install do |installer|
     target.build_configurations.each do |config|
       config.build_settings.delete 'IPHONEOS_DEPLOYMENT_TARGET'
       config.build_settings['ENABLE_BITCODE'] = 'NO'
+      # Fix some pods creating bunndle, asking for signing
```

Comment:
```suggestion
      # Fix some pods creating bundle, asking for signing
```

Reply:
Nice catch, time to change my keyboard switches 🙈